test/rust_crate_group/BUILD.bazel

```diff
@@ -0,0 +1,28 @@
+load("@rules_rust//rust:defs.bzl", "rust_crate_group", "rust_library")
+
+rust_library(
+    name = "dep1",
+    srcs = ["dep1.rs"],
+    edition = "2021",
+)
+
+rust_library(
+    name = "dep2",
+    srcs = ["dep2.rs"],
+    edition = "2021",
+)
+
+rust_crate_group(
+    name = "dep1_and_2",
+    deps = [
+        ":dep1",
+        ":dep2",
+    ],
+)
+
+rust_library(
```

Comment:
Could you also add a test to ensure `rust_test` targets are able to use the symbols from a "transitive" library?

Reply:
yes. done